src/lib/core.d.ts

```diff
@@ -836,7 +836,7 @@ interface RegExp {
     lastIndex: number;
 
     // Non-standard extensions
-    compile(): RegExp;
+    compile(): RegExp | undefined;

```

Comment:
Never mind, this is specced in ES6 annex B as returning RegExp always, and Chrome's implementation is wrong. I will revert this.